src/Tools/DynamoInstallDetective/AscSDKWrapper.cs

```diff
@@ -156,9 +157,29 @@ public ASC_STATUS GetInstalledPath(ref string installedPath)
                 return ASC_STATUS.INITIALIZE_FAILED;
             }
 
+            if (!string.IsNullOrEmpty(installPath))
+            {
+                installedPath = installPath;
+                return ASC_STATUS.SUCCESS;
+            }
+
             var status = ReadASCVersionFromEnv() == ASC_STATUS.SUCCESS ? ASC_STATUS.SUCCESS : ReadASCInstallPathFromRegistry(majorRelease);
-            installedPath = installPath;
+            if (status == ASC_STATUS.SUCCESS)
+            {
+                try
+                {
+                    string currentPath = Environment.GetEnvironmentVariable("PATH", EnvironmentVariableTarget.Process);
+                    string updatedPath = currentPath + ";" + installPath;
+                    Environment.SetEnvironmentVariable("PATH", updatedPath, EnvironmentVariableTarget.Process);
+                }
+                catch
+                {
+                    Trace.WriteLine("Failed to update path for ASC component");
```

Comment:
does trace make it to the dynamo log?

Reply:
no, trace does not make it to dynamo log.